cypress/tests/functional/Classic.spec.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file cypress/tests/functional/Classic.spec.js
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ */
+
+describe('Theme plugin tests', function() {
+	const journalPath = 'publicknowledge';
+	const index = 'index.php';
+	const path = '/' + index + '/' + journalPath;
+	const pages = [
+		'issue/current',
+		'issue/archive',
+		'issue/view/1',
+		'article/view/17',
+		'article/view/17/3',
+		'about',
+		'about/editorialTeam',
+		'about/submissions',
+		'about/contact',
+		'about/privacy',
+		'information/readers',
+		'information/authors',
+		'information/librarians'
+	];
+	const [month, day, year] = new Date()
+		.toLocaleDateString('en-US', {
+			month: 'numeric',
+			day: 'numeric',
+			year: 'numeric'
+		})
+		.split('/');
+	const user = {
+		'givenName': 'John',
+		'familyName': 'Debreenik',
+		'username': 'jdebreenik',
+		'country': 'UA',
+		'affiliation': 'Lorem Ipsum University'
+	};
+
+	it('Enables and selects the theme', function() {
+		cy.login('admin', 'admin', 'publicknowledge');
+
+		cy.get('nav[class="app__nav"] ul a[class="app__navItem"]:contains("Website")').click();
+		cy.get('button[id="plugins-button"]').click();
+
+		// Find and enable the plugin
+		cy.get('input[id^="select-cell-classicthemeplugin-enabled"]').click();
+		cy.get('div:contains(\'The plugin "Classic Theme" has been enabled.\')');
+		cy.reload();
+
+		// Select the Classic theme
+		cy.get('button[id="appearance-button"]').click();
+		cy.get('select[id="theme-themePluginPath-control"]').select('classic');
+		cy.get('div[id="theme"] button').contains('Save').click();
+		cy.get('div[id="theme"] [role="status"]').contains('Saved');
```

Comment:
Again, the less you specify the less likely you'll need to update the tests when there are syntax changes to the templates. Here I'd use:

```
cy.get('#theme [role="status"]').contains('Saved');
```

Reply:
Interesting that this line sometimes fails locally. This happens the first time this script is run after OJS installation, I've restarted the test to see if it's the case for Travis integration too.

Reply:
My mistake, this one: `cy.get('div:contains(\'The plugin "Classic Theme" has been enabled.\')')` after theme plugin is enabled